src/Game/GamePage.js

```diff
@@ -94,18 +109,37 @@ const GamePage = (props) => {
             setAttackTarget={setAttackTarget}
             totalPlayerData={totalPlayerData}
             playerName={playerName}
+            currentPlayerLocationID={currentPlayerLocationID}
           />
           <AttackButton
             totalPlayerData={totalPlayerData}
             roomCode={roomCode}
             attackTarget={attackTarget}
             playerName={playerName}
             isPlayerEliminated={isPlayerEliminated}
-            setTotalPlayerData={setTotalPlayerData}
             currentTurnPlayerIndex={currentTurnPlayerIndex}
+            currentTurnPlayer={currentTurnPlayer}
+            hasPlayerMovedLocations={hasPlayerMovedLocations}
+            setHasPlayerMovedLocations={setHasPlayerMovedLocations}
+          />
+          <PlayerLocations
+            playerName={playerName}
+            setCurrentPlayerLocationID={setCurrentPlayerLocationID}
+            currentTurnPlayer={currentTurnPlayer}
+            setHasPlayerMovedLocations={setHasPlayerMovedLocations}
+            hasPlayerMovedLocations={hasPlayerMovedLocations}
```

Comment:
might make sense to move the ui so it follows the order they would use it in. as in the location changing button on top of the attack stuff